src/index.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable no-use-before-define */
+
+import './style.css';
+import { library, dom } from '@fortawesome/fontawesome-svg-core';
+import {
+  faPlus,
+  faTrashAlt,
+  faArrowAltCircleRight,
+} from '@fortawesome/free-solid-svg-icons';
+
+library.add(faPlus, faTrashAlt, faArrowAltCircleRight);
+dom.watch();
+
+const tasks = [
+  { description: 'memo 1', completed: false, index: 1 },
+  { description: 'memo 2', completed: true, index: 2 },
+  { description: 'memo 3', completed: false, index: 3 },
+];
+
+document.addEventListener('DOMContentLoaded', () => {
+  const todoListElement = document.getElementById('todo-list');
+  todoListElement.innerHTML = '';
+
+  const listHeader = createListHeader();
+  todoListElement.appendChild(listHeader);
+
+  const addTaskField = createAddTaskField();
+  todoListElement.appendChild(addTaskField);
+
```

Comment:
[OPTIONAL] Good job working on the project requirements. But I noticed you disabled eslint in your index.js file it is against microverse policy to disable any linters because linters help us write clean code. So it would be nice if you enabled eslint. This should not be ignored I marked this as optional because the previous code reviewer did not spot this issue.